File: program/find-perimeter-of-a-rectangle/find_perimeter_of_a_rectangle.pl
#!/usr/bin/perl
use warnings;
use strict;
print "Enter the length : ";
my $length = <STDIN>;
print "Enter the width : ";
my $width = <STDIN>;
my $result = 2 * ($length+$width);
print "Perimeter of rectangle : ".$result. "\n";

File: program/find-perimeter-of-a-rectangle/find_perimeter_of_a_rectangle.dart
import 'dart:io';
void main(){
  print("Enter the length :");
  var length = int.parse(stdin.readLineSync());
  print("Enter the width :");
  var width = int.parse(stdin.readLineSync());
  var perimeter =  2 * (length + width);
  print('Perimeter of rectangle  : $perimeter');
}
